Update supported colors documentation with better explanation (#795)
okkur/syna: exampleSite/content/docs/supported-colors/content.md

| light     | #f8f9fa |
| dark      | #343a40 |

> For example, in order to change the primary color, you can update these line (or add if it doesn't exist) your `config.toml` file:

Reviewer: ```suggestion
In order to change the primary color, you can update the primary color within your `config.toml` file:
```

```toml
[params]
  [params.colors]
      primary = "#00838F"
```

Please note that `colors` is under `params` section of your configuration.

These colors are mainly used for backgrounds and button colors.  

> By changing the colors, all fragment colors and background will be changes to the new colors. Links, inline code and other elements will also be changed.

Reviewer: ```suggestion
By changing the colors, all fragment colors and backgrounds will be changed to the new colors. Links, inline code and other elements will also be changed.
```
